### ISSUE TITLE
Documentation: installing TruffleRuby in JVM mode from RVM

### ISSUE THREAD

**Issue by mohamedhafez** [oracle/truffleruby#3614]
In the Readme, it does not specify how to install TruffleRuby in JVM mode from RVM. If that's not supported, perhaps the Readme could mention that?

**Comment by eregon** [comment]
Yes, RVM does not support TruffleRuby in JVM mode currently and given the [complexity of RVM](https://eregon.me/blog/2021/06/04/review-of-ruby-installers-and-switchers.html#rvm) I didn't add it.
Although of course someone else could try adding it if they feel like it.

I don't really want to add this in the README because that section is meant to be concise.
It could be added in https://github.com/oracle/truffleruby/blob/master/doc/user/ruby-managers.md though.

IMO there are [few reasons](https://eregon.me/blog/2021/06/04/review-of-ruby-installers-and-switchers.html) to use RVM these days.

You should be able to install truffleruby manually via these instructions:
https://github.com/oracle/truffleruby/blob/master/doc/user/installing-truffleruby.md
That can probably also be added to rvm via `rvm mount` or so, but rvm adds a bunch of overhead and extra stuff that I wouldn't recommend.
